app/helpers/ethnic_background_helper.rb

```diff
@@ -0,0 +1,51 @@
+module EthnicBackgroundHelper
+  ETHNIC_GROUPS = [
+    'Asian or Asian British',
+    'Black, African, Black British or Caribbean',
+    'Mixed or multiple ethnic groups',
+    'White',
+    'Another ethnic group',
+  ].freeze
+
+  ETHNIC_BACKGROUNDS = {
+      'Asian or Asian British' => %w[Bangladeshi Chinese Indian Pakistani],
+      'Black, African, Black British or Caribbean' => %w[African Carribean],
+      'Mixed or multiple ethnic groups' => ['Asian and White', 'Black African and White', 'Black Caribbean and White'],
+      'White' => ['British, English, Northern Irish, Scottish, or Welsh', 'Irish', 'Irish Traveller or Gypsy'],
+      'Another ethnic group' => %w[Arab],
+    }.freeze
+
+  OTHER_ETHNIC_BACKGROUNDS = {
+    'Asian or Asian British' => ['Another Asian background', 'Your Asian background (optional)'],
+    'Black, African, Black British or Caribbean' => ['Another Black background', 'Your Black background (optional)'],
+    'Mixed or multiple ethnic groups' => ['Another Mixed background', 'Your Mixed background (optional)'],
+    'White' => ['Another White background', 'Your White background (optional)'],
+    'Another ethnic group' => ['Another ethnic background', 'Describe your ethnic background (optional)'],
+  }.freeze
```

Comment:
An alternative way to structure this is:

```ruby
ETHNIC_GROUPS = {
  'Asian or Asian British' => {
    backgrounds: %w[Bangladeshi Chinese Indian Pakistani],
    other: ['Another Asian background', 'Your Asian background (optional)']
  },
  ...
}

ETHNIC_BACKGROUNDS = do_stuff(ETHNIC_GROUPS)

OTHER_ETHNIC_BACKGROUNDS = do_other_stuff(ETHNIC_GROUPS)
```

Then we can spin off `ETHNIC_BACKGROUNDS` and `OTHER_ETHNIC_BACKGROUNDS` as transformations of the first `ETHNIC_GROUPS` hash.

But I think this could end up looking more complicated, and the only upside is that it DRYs up `'Asian or Asian British'` and the other keys. I don't expect these to change often, so the current solution I think is good!